### DISCUSSION TITLE
RAK support for POE

### DISCUSSION THREAD

**Discussion by ocgltd** [richonguzman/LoRa_APRS_iGate#166] [General]
I'm looking for an iGate board that can be powered off POE.  I found the RAK 13800 (https://store.rakwireless.com/products/rak13800-wisblock-ethernet-interface) plus POE board (https://store.rakwireless.com/products/wiscore-esp32-module-rak11200)

If I use their ESP32 core (https://store.rakwireless.com/products/wiscore-esp32-module-rak11200) will the firmware work?  (Since it's ESP32) or does the firmware need to be certified for a specific board?

If it won't work, anyone got an idea to run an igate off POE?

**Comment by richonguzman** [comment]
Hi, 

I've never tested those boards to know if this will work... so for now I cant answer .

**Comment by dl1lmh** [comment]
Just found this discussion... 

I had the same idea and just implemented it.

Works perfectly in my home. I used an OLIMEX ESP32-POE-ISO board and a self-made LoRa board.

Maybe this is interesting for you, take a look at pull request #192.

**Reply by richonguzman** [reply]
I saw the PR and looking for reading it in detail... may I write you directly over telegram or some sort of Messenger ?

**Reply by dl1lmh** [reply]
Yes of course!

**Reply by dl1lmh** [reply]
Had to figure out my Username: Pls try "KampfKeks_MRZ"
